cypress/support/index.js

```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
+// This example support/index.js is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
+// behavior that modifies Cypress.
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import './commands'
```

Comment:
Where is this running browser or Node.js?

Reply:
Removed this file as it was an example file and I don't need it: https://github.com/commercetools/commercetools-adyen-integration/pull/28/commits/c4a66ac27af1683c82579dd00782a326aeaa4776